### PR TITLE
fix(outputs.sql): Enable the use of krb5 with mssql driver

### PR DESCRIPTION
**Pull request by powersj**
## Checklist

- [x] No AI generated code was used in this PR

## Related issues

fixes: #15194

### CODE REVIEW

**Comment by vin-ediezca** [comment]
@powersj linux_amd64.tar.gz image does not have `sh` or `bash` shells. I have to install some KRB5 dependencies first to test. do you have an artifact that has any of two?

**Comment by vin-ediezca** [comment]
Test passed @powersj. Thank you so much!

image

**Comment by powersj** [comment]
Thanks for confirming!

File: go.mod [hunk 1]
```diff
@@ -224,6 +224,8 @@ require (
 	modernc.org/sqlite v1.29.5
 )
 
+require github.com/jcmturner/goidentity/v6 v6.0.1 // indirect
```
Comment (DStrand1) [inline go.mod:227]: Could you move this into the main indirect `require` block?

**Review by DStrand1 — APPROVED** [review]
LGTM! Just one small comment

**Review by srebhan — APPROVED** [review]
LGTM. Can we please also add this to the `inputs.sql` plugin in a separate PR!?!?

**Pull request merged**